bundles/AdminBundle/Resources/public/js/pimcore/object/tags/advancedManyToManyObjectRelation.js

```diff
@@ -131,9 +131,15 @@ pimcore.object.tags.advancedManyToManyObjectRelation = Class.create(pimcore.obje
                 fc.layout = field;
                 fc.editor = null;
                 fc.sortable = false;
-                if(fc.layout.key === "fullpath") {
+
+                if (fc.layout.key === "fullpath") {
                     fc.renderer = this.fullPathRenderCheck.bind(this);
+                } else if(fc.layout.layout.fieldtype == "select" || fc.layout.layout.fieldtype == "multiselect") {
```

Comment:
please fix it for grid view as well..thanks! 
![image](https://user-images.githubusercontent.com/5137917/93747220-aeb75b00-fbf6-11ea-8a88-9db2cea81d86.png)

somewhere here: https://github.com/pimcore/pimcore/blob/e6f3de97c326595166907595a56e33176aa2cc11/bundles/AdminBundle/Resources/public/js/pimcore/object/helpers/grid.js#L406

Reply:
ping @skochTC

Reply:
Yeah, sure. We will adapt this to the grid view.

But we have a lot to do this week.

Reply:
I've changed this, it's now also working for a grid-view.